### ISSUE TITLE
Postgresql: ERROR: procedure merge_knownexploited(character varying, ...) does not exist

### ISSUE THREAD

**Issue by Tord-BG** [dependency-check/DependencyCheck#5379]
**Describe the bug**
When using version 8.0.1 and a postgres DB we get the exception stated in the subject. The DB is initialized with the latest script supplied for postgres with no errors showing when running the script.

**Version of dependency-check used**
The problem occurs using version 8.0.1 of the maven-plugin

**Log file**
An exception occurred while downloading updates. Please see the log file for more details. java.sql.BatchUpdateException: Batch entry 0 CALL merge_knownexploited('CVE-2021-27104','Accellion','FTA','Accellion FTA OS Command Injection Vulnerability','2021-11-03','Accellion FTA 9_12_370 and earlier is affected by OS command execution via a crafted POST request to various admin endpoints.','Apply updates per vendor instructions.','2021-11-17',NULL) was aborted: ERROR: procedure merge_knownexploited(character varying, character varying, character varying, character varying, character varying, character varying, character varying, character varying, character varying) does not exist
[ERROR]   Hint: No procedure matches the given name and argument types. You might need to add explicit type casts.

**To Reproduce**
1. Init postgres DB with script supplied for 8.0.1
2. Run version 8.0.1 with the updateOnly-goal

Any help with this is would be much appreciated!

**Comment by jeremylong** [comment]
Can you double check that you are using the latest DB initialization script for postgres? The procedure was introduced with 8.0.1.

https://github.com/jeremylong/DependencyCheck/blob/496ffc689ae588eea6e96a60c2ce02c9ca0fe518/core/src/main/resources/data/initialize_postgres.sql#L108

**Comment by Tord-BG** [comment]
You pointed me in the right direction there. That script gave me error now when it wants to drop the procedure if it already exists. Turned out our DB also contained a function with that name but with different arguments so I must have used a script somewhere in the repo that added that one. 

So when I manually dropped both of these functions and then ran the init-script again everything started working as expected. 

Thank you for your quick assistance !
